release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -242,6 +242,11 @@ For more information, see xref:../scalability_and_performance/what-huge-pages-do
 [id="ocp-4-8-scale"]
 === Scale
 
+[id="ocp-4-8-performance-creator-tool"]
+==== Creating a performance profile
+You can now create a performance profile using the Performance Profile Creator (PPC) tool. The tool consumes `must-gather` data from the cluster, several user-supplied profile arguments, and generates a performance profile that is appropriate for your hardware and topology.
```

Comment:
This sentence: 

> The tool consumes `must-gather` data from the cluster, several user-supplied profile arguments, and generates a performance profile that is appropriate for your hardware and topology.

It treats the three items in the sentence as if they are all the kind of list item in a series. However, it seems like the actual tool is gathering data and arguments, and generating a performance profile. I would maybe reword it a little bit?

Reply:
HI @sagidlow was going to adjust to this 
The tool consumes `must-gather` data from the cluster, several user-supplied profile arguments, and using this information generates a performance profile that is appropriate for your hardware and topology.

The tool itself does not gather the data it uses the data captured in must-gather with various command-line arguments to generate a profile.

Reply:
I would suggest rewording to something like this:

> The tool consumes `must-gather` data from the cluster and several user-supplied profiles arguments, and using this information it generates a performance profile that is appropriate for your hardware and topology. The tool itself does not gather the data. It uses the data captured in `must-gather` with various command line arguments to generate a profile.

Reply:
sorry was not clear I think to keep it simple I will adjust to this "The tool consumes must-gather data from the cluster, several user-supplied profile arguments, and using this information generates a performance profile that is appropriate for your hardware and topology." I am trying to keep RN information quite minimal. They can read more in the link.

Reply:
My apologies I thought the "tool itself" portion was part of the text you wanted to add to the RN. The suggestion below for the first sentence. There are still some minor grammar issues for the initial list so if you could please address that. 

> The tool consumes `must-gather` data from the cluster and several user-supplied profiles arguments, and using this information it generates a performance profile that is appropriate for your hardware and topology.